### ISSUE TITLE
Add useNativeDriver as an option for Animation

### ISSUE THREAD

**Issue by alex4dev** [crazycodeboy/react-native-easy-toast#80]
Hi, i've seen useNativeDriver is not set for Animated api, why?
Maybe we can add a prop in Toast component for that.

Thanks

**Comment by alex4dev** [comment]
My pull request her : [#81 ](https://github.com/crazycodeboy/react-native-easy-toast/pull/81)

**Comment by pabloluz** [comment]
@crazycodeboy can we have this merged, please?

**Comment by vinodhram3** [comment]
I am also facing the same issue.

**Comment by ErickArciniega** [comment]
Yes I have the same issue, it's necessary implement useNativeDriver for smother animations, any solution?

**Comment by Alejandrehl** [comment]
Any solution?

**Comment by NathanNovak** [comment]
> Any solution?

I went into the node modules and added it myself. I noticed this repo has been forked over 200 times so I assume that the creator is no longer maintaining it.

**Comment by pabloluz** [comment]
I did the same. For everyone else who is looking for a (palliative) solution, you guys can simply do the same as #81. It's super straightforward ;)

**Comment by ArauzVega** [comment]
Someone who fixed it did so as follows: 
 node_modules package> react native-easy-toast> index.js 
and add the following configuration:

In show function:

![image](https://user-images.githubusercontent.com/42548300/88722296-39db0f80-d0e4-11ea-9bc1-a71907ad5152.png)

In close function:

![image](https://user-images.githubusercontent.com/42548300/88722432-773f9d00-d0e4-11ea-951c-4cce11df8e6d.png)

**Comment by Bitenca** [comment]
Guys? anybody can merge pls?

**Comment by ArauzVega** [comment]
> Guys? anyone can merge pls?

You can try this too :)

![image](https://user-images.githubusercontent.com/42548300/90273127-b4619a00-de1b-11ea-8185-78bc1590c17e.png)
